pythonforandroid/prerequisites.py

```diff
@@ -96,11 +93,45 @@ def darwin_helper(self):
     def linux_helper(self):
         info(f"No helper available for prerequisite: {self.name} on linux")
 
+    def _darwin_get_brew_formula_location_prefix(self, formula, installed=False):
+        opts = ["--installed"] if installed else []
+        p = subprocess.Popen(
+            ["brew", "--prefix", formula, *opts],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        _stdout_res, _stderr_res = p.communicate()
+
+        if p.returncode != 0:
+            error(_stderr_res.decode("utf-8").strip())
+            return None
+        else:
+            return _stdout_res.decode("utf-8").strip()
+
+
+class HomebrewPrerequisite(Prerequisite):
+    name = "homebrew"
+    mandatory = dict(linux=False, darwin=True)
+    installer_is_supported = dict(linux=False, darwin=False)
+
+    def darwin_checker(self):
+        if shutil.which("brew"):
+            return True
+        else:
+            return False
```

Comment:
```suggestion
        return shutil.which("brew")
```

Reply:
I'm expecting (here and in similar ones), to need a "clean" `boolean` in future, so returning the `sh.which("brew")` ouput which is a string doesn't seem a good idea.

How about:
```python
return shutil.which("brew") != ""
```

Reply:
Oh yeah my bad, you're right.
Yeah in this case you can either:
```python
return shutil.which("brew") is not None
```
Or:
```python
return bool(shutil.which("brew"))
```